README.md

```diff
@@ -0,0 +1,15 @@
+# Biological ontologies as hetnets in Neo4j
+
+This repository creates a Neo4j database of prominent biological ontologies. The import is accomplished using the EBI's [`ols-neo4j-app`](https://github.com/EBISPOT/OLS/tree/master/ols-apps/ols-neo4j-app "EBISPOT/OLS on GitHub"). The compressed Neo4j database ([`hetontology.db.tar.xz`](database/hetontology.db.tar.xz)) can be easily deployed to a Neo4j instance to bring the convenience of the Cypher query language to ontologies.
```

Comment:
Might be helpful to specify if it includes the ontology terms *and* the annotations or just the terms.